src/test/java/core/basesyntax/database/StorageDaoImplTest.java

```diff
@@ -0,0 +1,81 @@
+package core.basesyntax.database;
+
+import core.basesyntax.model.Fruit;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.NoSuchElementException;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class StorageDaoImplTest {
+    private static StorageDaoImpl storageDao;
+    private static List<Fruit> fruitList;
+    private Fruit fruit;
+
+    @BeforeClass
+    public static void beforeClass() throws Exception {
+        storageDao = new StorageDaoImpl();
+        fruitList = new ArrayList<>();
+
+    }
+
+    @Before
+    public void setUp() throws Exception {
+        fruit = new Fruit("apple");
+    }
+
+    @After
+    public void tearDown() throws Exception {
+        Storage.fruits.clear();
+    }
+
+    @Test
+    public void fruit_add_ok() {
```

Comment:
test method name should look like this: methodName_state_expectedBehavior
```suggestion
    public void add_correctFruitName_ok() {
```